<mention service name>/pom.xml

```diff
@@ -0,0 +1,449 @@
+<?xml version="<mention specific version required>" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/<mention specific version required>"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/<mention specific version required> http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion><mention specific version required></modelVersion>
+
+    <groupId>io.mosip.<mention service name></groupId>
+    <artifactId><mention service name>-parent</artifactId>
+    <version><mention specific version required>-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name><mention service name></name>
+    <description>Parent project of MOSIP <mention service name></description>
+    <url>https://github.com/mosip/<mention service name></url>
+
+    <licenses>
+        <license>
+            <name>MPL <mention specific version required></name>
+            <url>https://www.mozilla.org/en-US/MPL/<mention specific version required>/</url>
+        </license>
+    </licenses>
+    <scm>
+        <connection>scm:git:git://github.com/mosip/<mention service name>.git</connection>
+        <developerConnection>scm:git:ssh://github.com:mosip/<mention service name>.git</developerConnection>
+        <url>https://github.com/mosip/<mention service name></url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <developers>
+        <developer>
+            <name>Mosip</name>
+            <email>mosip.emailnotifier@gmail.com</email>
+            <organization>io.mosip</organization>
+            <organizationUrl>https://github.com/mosip/<mention service name></organizationUrl>
+        </developer>
+    </developers>
+
```

Comment:
Please check with chandra regarding the url used

Reply:
change the Licence to MPL2

Reply:
organizationUrl:  https://github.com/mosip   --> https://mosip.io

Reply:
<url>https://github.com/mosip/<mention service name></url>

to

<url>https://mosip.io<mention service name></url>